### PR TITLE
fixes dead link on blog

### PR DESCRIPTION
**Pull request by rmariani**

### CODE REVIEW

File: content/blog/bootstrapping-urbit-from-ethereum.md [hunk 1]
```diff
@@ -378,10 +378,9 @@ price in ether of a planet, and the number of planets available.
 
 ## Okay, so
 
-That was quite a lot of information!  We've created a fora thread
-[here](https://urbit.org/fora/posts/~2017.9.20..22.56.04..47ce~) for discussion,
-and would be glad to hear your feedback.  GitHub issues on the
-[urbit/constitution repo](https://github.com/urbit/constitution) are also more
+That was quite a lot of information! There's also a blog post about the constitution
+[here](https://urbit.org/blog/interim-constitution/), and GitHub issues on the
+[urbit/constitution repo](https://github.com/urbit/constitution) are more
```
Comment (matildepark) [inline content/blog/bootstrapping-urbit-from-ethereum.md]: This link is also dead — links to a private repo. Where does our actual constitution information live now?

Comment (rmariani) [reply content/blog/bootstrapping-urbit-from-ethereum.md]: Perhaps we could just make the repo public? @galenwp what do you think?

Comment (baudtack) [reply content/blog/bootstrapping-urbit-from-ethereum.md]: @galenwp Ping.

Comment (galenwp) [reply content/blog/bootstrapping-urbit-from-ethereum.md]: @Fang- will know which repo is correct. I think https://github.com/urbit/azimuth ? We called it the constitution for a while before changing it.

Comment (baudtack) [reply content/blog/bootstrapping-urbit-from-ethereum.md]: @Fang- ping

Comment (Fang-) [reply content/blog/bootstrapping-urbit-from-ethereum.md]: Azimuth is indeed the obviously correct repo here.

**Review by baudtack — APPROVED** [review]

**Pull request merged**